lib/fs-wrapper.js

```diff
@@ -90,7 +89,7 @@ module.exports = {
 
   readdir: fsp.readdir,
 
-  remove: rimraf,
+  remove: fsp.rm,
```

Comment:
I imagine we need to turn on the `recursive` option on by default?
https://nodejs.org/api/fs.html#fspromisesrmdirpath-options

Reply:
Yeah, probably.
It's deprecated, not sure why.

Reply:
Oh, rmdir recursive is deprecated in favor of rm recursive.